docs/system_overview/configuration.md

```diff
@@ -33,6 +47,8 @@ This file contains the following top level entries:
 
     - `transfer_chunk_size_for_parallel_transfer_in_megabytes` (required) (default 40)
 
+  - `client_api_whitelist_policy` (default "") - If set to "enforce", the server will limit the scope of operations available to non-administrative clients.
```

Comment:
empty string is default?

Reply:
I'm confused about what the context is around _default_. Is it for vanilla installs or when the option is not explicitly defined due to an upgrade?

Reply:
i think we're talking here about a vanilla installation.    upgrade (undefined) considerations are subtly different, agreed.

Reply:
ok. i'll set it to "enforce"